apps/demo-game/prisma/migrations/20240827111207_countdown_duration/migration.sql

```diff
@@ -0,0 +1,18 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `avatar` on the `Player` table. All the data in the column will be lost.
+  - You are about to drop the column `color` on the `Player` table. All the data in the column will be lost.
+  - You are about to drop the column `location` on the `Player` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "PeriodSegment" ADD COLUMN     "countdownDurationMs" INTEGER,
+ADD COLUMN     "countdownExpiresAt" TIMESTAMP(3);
+
+-- AlterTable
+ALTER TABLE "Player" DROP COLUMN "avatar",
```

Comment:
why is the player table altered on this pr (or why is there a migration but no change in schema)? did we forget to do a migration on another branch?

Reply:
maybe that stuff was moved into facts but we didn't do the migration yet?

Reply:
This should be fixed now as we discussed together.